pagopa/io-app
# feat(Sicilia Vola): [IASV-35] Add ToS component

## Short description
This PR adds the Term of Services screen
The user should view and accept this screen in order to proceed to request the discount code

### ToS
https://user-images.githubusercontent.com/822471/126793617-4a17c272-2137-406e-8bd4-b75c8245e8fb.mov

### how to test
Since the entry point is not already implement, follow these steps
- edit a component is surely mounted (es `MessagesHomeScreen`)
- in its componentDidMount (or useEffect) method/function add these two Reactotron custom commands
```typescript
RTron?.onCustomCommand({
      title: "start Sicilia Vola",
      description: "start sicilia vola flow",
      command: "start Sicilia Vola",
      handler: () => {
        this.props.startSiciliaVola();
      }
    });
    RTron?.onCustomCommand({
      title: "Alive + Tos Not Accepted",
      description: "Alive + Tos Not Accepted",
      command: "Alive + Tos Not Accepted",
      handler: () => {
        this.props.aliveAndTosNotAccepted();
      }
    });
```
- in order to activate this screen you have to press the first command and then the secondo one (see the attached video)
